src/Adapters/Storage/Mongo/MongoTransform.js

```diff
@@ -1403,10 +1403,19 @@ const mongoObjectToParseObject = (className, mongoObject, schema) => {
           case 'times_used':
             restObject['timesUsed'] = mongoObject[key];
             break;
+          case 'authData':
+            if (className === '_User') {
+              log.warn(
```

Comment:
why not throw here?

Reply:
This case will only happen if someone directly changed the DB or if a change was made before this PR i.e authData was set to something (null).

I believe we can just ignore this without log, worst case scenario there will be an authData field in the database forever (unless they clean it up).

If we throw an error or log here, the developer would have to find all users and delete `authData: null` fields.

Best case we could do cleanup when the server starts.

I'll let you guys decide.

Reply:
don't let ME decide ;).  If you're good with this, so am i.

Reply:
I suggest logging. Removing unused fields reduces the DB working set size and makes the queries cheaper. In regards to effort: it's a simple `updateMany` to remove all `authData: null` fields.

For future reference:
`db.getCollection("_User").updateMany( { authData: {$type: 10} }, { $unset: { "authData": "" } } );`

Reply:
The next server release is 4.0.0. We can add `updateMany` to migration guide.